services/provider/server/server.go

```diff
@@ -146,7 +146,10 @@ func (s *OCSProviderServer) AcknowledgeOnboarding(ctx context.Context, req *pb.A
 		}
 		return nil, status.Errorf(codes.Internal, "Failed to update the storageConsumer. %v", err)
 	}
-
+	// create noobaa account CR
+	if err := s.consumerManager.CreateNoobaaAccount(ctx, req.StorageConsumerUUID); err != nil {
```

Comment:
It will be better to add a test case for creating and deleting the NoobaaAcoount. As we already have a separate function for it.

Reply:
Sure, will do that in a seperate PR.
Tracking here https://github.com/red-hat-storage/ocs-operator/issues/2710